### PR TITLE
[HOLD] Go live with save in progress for hca

### PR DESCRIPTION
**Pull request by jbalboni**
Just so we have the branch ready to go for tomorrow.

### CODE REVIEW

File: src/js/hca/config/form.js [hunk 1]
```diff
@@ -119,14 +118,13 @@ const formConfig = {
...
   // Disable save in progress for production
-  disableSave: __BUILDTYPE__ === 'production',
   savedFormErrorMessages: {
     notFound: 'Please start over to apply for health care.',
     noAuth: 'Please sign in again to resume your application for health care.'
   },
   transformForSubmit: transform,
   // Use the old intro page for production, but SiP for dev and staging
-  introduction: __BUILDTYPE__ === 'production' ? IntroductionPage : SIPIntroductionPage,
+  introduction: SIPIntroductionPage,
```
Comment (jbalboni) [inline src/js/hca/config/form.js]: I could move SIPIntroductionPage to IntroductionPage, but that would invite merge conflicts.

File: src/js/hca/config/form.js [hunk 2]
```diff
@@ -119,14 +118,13 @@ const formConfig = {
   formId: '1010ez',
   version: 0,
   // Disable save in progress for production
```
Comment (cvalarida) [inline src/js/hca/config/form.js]: Probably remove this comment as well.

File: src/js/hca/config/form.js [hunk 3]
```diff
@@ -119,14 +118,13 @@ const formConfig = {
...
-  disableSave: __BUILDTYPE__ === 'production',
   savedFormErrorMessages: {
     notFound: 'Please start over to apply for health care.',
     noAuth: 'Please sign in again to resume your application for health care.'
   },
   transformForSubmit: transform,
   // Use the old intro page for production, but SiP for dev and staging
```
Comment (cvalarida) [inline src/js/hca/config/form.js]: And this one.

**Review by cvalarida — APPROVED** [review]
🚀 When we're ready.

**Comment by jbalboni** [comment]
I added a minor change to remove some hca specific messaging from the saved form page.

**Review by cehsu — APPROVED** [review]

**Review by markolson — APPROVED** [review]

**Pull request merged**